### DISCUSSION TITLE
Can I keep the SSH key in memory?

### DISCUSSION THREAD

**Discussion by louwers** [maxgoedjen/secretive#474] [Q&A] [answered]
I don't want to reauthenticate all the time. Can the agent keep it in memory?

**Comment by martinpaljak** [comment, marked as answer]
From the popup notification, choose "keep unlocked" with the amount of time you want it to be usable without re-authentication

**Comment by chriszarate** [comment]
Tip: Make sure you have notifications for Secretive turned on in System Settings > Notifications > Application Notifications > Secretive. Authenticate, then find the resulting notification in the Notification Center. (I was confusing "popup notification" with the Touch ID authentication modal, which has no such "keep unlocked" option.)

**Reply by martinpaljak** [reply]
I wish it had, because the notification often disappears prematurely or just lacks context, and I get prompted again.
